packages/cli/lib/lib/babel-config.js

```diff
@@ -11,11 +11,12 @@ module.exports = function (env, options = {}) {
 					targets: {
 						browsers: options.browsers,
 					},
-					exclude: ['transform-regenerator', 'transform-async-to-generator'],
+					exclude: ['transform-regenerator'],
 				},
 			],
 		],
 		plugins: [
+			require.resolve('babel-plugin-transform-async-to-promises'),
```

Comment:
in modern mode we don't need this. All modern browsers support async/await.  Wouldn't adding this here add it for both modern and legacy mode?

Reply:
True. Removed.

Reply:
Hm, now the tests are failing again. I'll admit this task proved more complex than I thought so I would welcome some guidance around it.